wp-content/civi-extensions/goonjcustom/CRM/Goonjcustom/CivirulesAction/CreateEventForContact.php

```diff
@@ -24,6 +25,8 @@ public function processAction(CRM_Civirules_TriggerData_TriggerData $triggerData
 			'is_public' => 1,
 			'default_role_id' => 1,
 			'created_id' => $contactId,
+			'is_map' => 1,
```

Comment:
what does this do?

Reply:
@pokhiii
This will mark `Include Map to Event Location` to check. 
**Purpose:**
If your event location is provided (e.g., address), setting is_map to 1 helps display a map on the event’s public-facing page. This allows attendees to find the event location easily.

<img width="1193" alt="image" src="https://github.com/user-attachments/assets/552bf66a-059a-4588-9358-39afc46a483b">

I expect we don't need it right now because there is no requirement for this. Let me remove it.